src/pages/youthboard/surveys/index.tsx

```
@@ -0,0 +1,132 @@
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'next-i18next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+
+const surveys = () => {
+  const { t } = useTranslation();
+  const theme = useTheme<any>();
+  const router = useRouter();
+
+  const [value, setValue] = useState<number>(1);
```

ESLint

[error] React Hook "useState" is called in function "surveys" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)